src/ray/object_manager/object_directory.cc

```diff
@@ -13,27 +13,27 @@ using ray::rpc::GcsNodeInfo;
 using ray::rpc::ObjectTableData;
 
 /// Process a notification of the object table entries and store the result in
-/// client_ids. This assumes that client_ids already contains the result of the
+/// node_ids. This assumes that node_ids already contains the result of the
 /// object table entries up to but not including this notification.
 void UpdateObjectLocations(const GcsChangeMode change_mode,
                            const std::vector<ObjectTableData> &location_updates,
-                           const ray::gcs::ClientTable &client_table,
-                           std::unordered_set<ClientID> *client_ids) {
+                           std::shared_ptr<gcs::RedisGcsClient> gcs_client,
```

Comment:
Ok. But not now,  because there are still many other table in use, such as `ActorCheckpointTable`, `HeartbeatTable`, `DynamicResourceTable` and so on.  I can fix this after all the use of  table has been removed.